debezium-e2e-benchmark/py/tpc-run-test.py

```diff
@@ -0,0 +1,223 @@
+    try:
+        print(tpcconfig['jdbc'][config['config']
+                                ['connector.class'].split('.')[3]]['tpctable'])
+        curs.execute(tpcconfig['jdbc'][config['config']
+                                       ['connector.class'].split('.')[3]]['tpctable'])
+        conn.commit()
+        print('table created')
+        return 0
+    except:
+        print('table create error')
+        return 1
+
+
+def enablecdctablesql(conn, config, tpcconfig):
+    curs = conn.cursor()
+    for sql in tpcconfig['jdbc'][config['config']['connector.class'].split('.')[3]]['enablecdctablesql']:
+        if sql.startswith('python.time.sleep'):
+            print('Wait for ' + sql[17:] + ' second')
+            time.sleep(int(sql[17:]))
+        else:
+            try:
+                curs.execute(sql)
+                conn.commit()
+                print(sql)
+            except:
+                print(sql + ' done with exception  !!')
+    return 0
+
+
+def topicexport(bootstrapserver, count, commitinterval):
+    consumer = KafkaConsumer('db2server.TPC.TEST',
+                             bootstrap_servers=bootstrapserver,
+                             auto_offset_reset='smallest',
+                             enable_auto_commit=True,
+                             )
+    i = 0
+    file = open("/home/tpc/tpcdata/" + "tpc_" + str(count) +
+                "_" + str(commitinterval) + ".csv", "w")
+    print('read from TOPIC')
+    for message in consumer:
+        # message value and key are raw bytes -- decode if necessary!
+        # e.g., for unicode: `message.value.decode('utf-8')`
+        print("%s:%d:%d: key=%s value=%s" % (message.topic, message.partition,
+                                             message.offset, message.key,
+                                             message.value))
+
+        d = json.loads(message.value)
+        file.write(str(message.timestamp) + "000;" + str(d['payload']['after']['T0']) +
+                   ";" + d['payload']['op'] + ";" + str(i) + ";" + str(d['payload']['after']['ID']) + "\n")
+        i = i + 1
+        print(i)
+        print(count)
+        if i >= count:
+            break
+    file.close()
+
+
+def getjdbcconnection(config, tpcconfig):
+    connectiontype = config['config']['connector.class'].split('.')[3]
+    jdbctype = 'jdbc:' + connectiontype + '://'
+    if connectiontype == 'oracle':
+        jdbctype = 'jdbc:oracle:thin:@'
+    conn = jaydebeapi.connect(tpcconfig['jdbc'][connectiontype]['jdbcdriver'], jdbctype +
+                              config['config']['database.hostname'] + ':' +
+                              config['config']['database.port'] + '/' +
+                              config['config']['database.dbname'],
+                              [config['config']['database.user'],
+                                  config['config']['database.password']],
+                              '/home/tpc/jdbcdriver/' + tpcconfig['jdbc'][connectiontype]['jar'])
```

Comment:
Hardocded path

Reply:
It is hardcoded because running in the docker. It works in the same way for "Benchmark on the existing environment (DB Server / Kafka / Connector)" and "Benchmark in a self-contained TPC set up for DB2 with Docker compose"

Reply:
Makes sense, still I'd like to see it as a constant in the file, not buried in the code.

Reply:
as global declared